rules/figma-prototypes/rule.md

```diff
@@ -0,0 +1,42 @@
+---
+type: rule
+title: Do you build interactive prototypes in Figma?
+uri: figma-prototypes
+authors:
+  - title: Joseph Fernandez
+    url: https://ssw.com.au/people/joseph-fernandez/
+created: 2023-12-15T04:29:01.737Z
+guid: a0836b52-0b01-44e5-a110-8c7496b1a7a9
+---
+Figma is a powerful design tool that offers several prototyping features that are valuable for UI designers. Here are a few of the top Figma prototyping features:
+
+`youtube: https://youtu.be/-d6zNGeF59M?si=dYJCN_Hf3zKXiHTX`
+**Video: Figma Tutorial: Prototyping & Transitions (5 min)**
+
+## Interactive Components:
+
+Figma's Interactive Components feature allows UI designers to easily create dynamic and interactive user interfaces. You can define different states for a component, such as hover or pressed states, and make transitions between these states. This helps simulate how users interact with the final product, providing a more realistic representation of the user experience.
```

Comment:
I've updated the text but kept the keyword "Such as" only because it implies that these are just suggested terminologies that can be changed. We are also not restricted to just creating 4 different states.